CMakeSettings.json

```diff
@@ -104,6 +104,10 @@
       "cmakeCommandArgs": "",
       "buildCommandArgs": "-v -j8",
       "variables": [
+        {
+          "name": "ENABLE_LIBMODBUS",
+          "value": "ON"
```

Comment:
This seems to be out of sync with the option in cmakelists.txt, where it defaults to off.

Reply:
This is a different environment and specifies a build, so he is enabling it here.